res/css/views/settings/_LayoutSwitcher.pcss

```diff
@@ -15,79 +15,74 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-.mx_LayoutSwitcher_RadioButtons {
+.mx_LayoutSwitcher_LayoutSelector {
     display: flex;
-    flex-direction: row;
-    gap: 24px;
-    width: 100%;
+    flex-direction: column;
+    gap: var(--cpd-space-4x) !important;
 
-    color: $primary-content;
+    .mxLayoutSwitcher_LayoutSelector_LayoutRadio {
+        border: 1px solid var(--cpd-color-border-interactive-primary);
+        border-radius: var(--cpd-space-2x);
 
-    > .mx_LayoutSwitcher_RadioButton {
-        flex-grow: 0;
-        flex-shrink: 1;
-        display: flex;
-        flex-direction: column;
-        overflow: hidden;
-
-        flex-basis: 33%;
-        min-width: 0;
-
-        border: 1px solid $quinary-content;
-        border-radius: 10px;
-
-        .mx_EventTile_msgOption,
-        .mx_MessageActionBar {
-            display: none;
-        }
-
-        .mx_LayoutSwitcher_RadioButton_preview {
-            flex-grow: 1;
+        .mxLayoutSwitcher_LayoutSelector_LayoutRadio_inline {
             display: flex;
+            gap: 10px;
```

Comment:
cpd token maybe?

Reply:
There is no `10px` cpd token but the gap is `10px` in the figma.